docs/book/src/developer/providers/migrations/v1.10-to-v1.11.md

```diff
@@ -20,6 +20,11 @@ proposal because most of the changes described below are a consequence of the wo
     - the old `status.conditions` will continue to exist temporarily under `status.deprecated.v1beta1` for the sake of
       down conversions and to provide a temporary option for users willing to continue using old conditions.
   - Support for terminal errors has been dropped from all Kinds
+    - The `Failed` phase has been removed from the enum values for:
```

Comment:
Is this a problem for conversion?

(Maybe we have to keep the enum values until we removed v1beta1 and don't need conversion anymore?)

Reply:
I have restored values, so we are on the safe side